src/components/Calendar.vue

```diff
@@ -1,13 +1,45 @@
 <template>
-  <div>
-    <h1>Calendar View</h1>
-  </div>
+  <v-container fluid>
+    <full-calendar :events="fcEvents"/>
+  </v-container>
 </template>
 
 <script>
-export default {
 
+export default {
+  components: {
+    'full-calendar': require('vue-fullcalendar')
+  },
+  computed: {
+    fcEvents () {
+      const Slides = this.$store.getters.getSlide
+      const filteredSlides = (Slides.map((slide) => {
+        const newSlide = {}
```

Comment:
I would suggest renaming this to event, because the FC takes a list of events

Reply:
fixed